### ISSUE TITLE
Unable to enable accessibility on macOS

### ISSUE THREAD

**Issue by snowe2010** [rvaiya/warpd#64]
started warpd, log says

```
❯ tail -f /Users/tyler/.config/warpd/warpd.log
Starting warpd: 1.0.2-beta
Failed to create event tap, make sure warpd is whitelisted as an accessibility feature.
```

Enabled the accessibility feature (it didn't show up in the list, I had to add it manually, pointed at `/usr/local/bin/warpd`) restarted, and still have the same error. 

![image](https://user-images.githubusercontent.com/1534535/160923062-dd95f12a-e4c2-4e67-92fb-b2a2a2a8edef.png)

Do I need to enable this in a different location as well?

**Comment by snowe2010** [comment]
Solved this (partially) by running `sudo tccutil reset Accessibility` which promptly crashed my computer. I hard reset, it asked for accessibility for all my apps, it had removed the warpd option that I had added manually, I added it myself again, and then started warpd and it worked!

**Comment by snowe2010** [comment]
I think it would probably be best (this is now a feature request) to implement something like this, https://stackoverflow.com/questions/35605949/relaunch-os-x-app-to-execute-cgeventtapcreate-when-axisprocesstrusted-is-tru, which continually checks until you have permission and then relaunches the app.

**Comment by rvaiya** [comment]
Thanks for reporting this. I believe it should be fixed in the latest commit.
